subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -1,12 +1,22 @@
 [[declaring-repositories]]
 = Declaring repositories
 
-Gradle can resolve dependencies from one or many repositories based on Maven, Ivy or flat directory formats. Check out the <<#sec:repository-types,full reference on all types of repositories>> for more information.
+Gradle can resolve dependencies from one or many repositories based on Maven, Ivy or flat directory formats.
+Check out the <<#sec:repository-types,full reference on all types of repositories>> for more information.
 
-[[sec:frequently_used_repositories]]
-== Frequently used repositories
+[[sec:declaring_public_repository]]
+== Declaring a publicly-available repository
+
+Organizations building software may want to leverage public binary repositories to download and consume open source dependencies.
+Popular public repositories include <<#sub:maven_central,Maven Central>>, <<#sub:maven_jcenter,Bintray JCenter>> and the <<#sub:maven_google,Google Android>> repository.
+Gradle provides built-in shortcut methods for these widely-used repositories.
+
+.Declaring a repository with the help of shortcut methods
+image::dependency-management-shortcut-repositories.png[]
 
-Gradle supports the most frequently used Java repositories out of the box.
+Under the covers Gradle resolves dependencies from the respective URL of the public repository defined by the shortcut method.
+All shortcut methods are available via the link:{groovyDslPath}/org.gradle.api.artifacts.dsl.RepositoryHandler.html[RepositoryHandler] API.
```

Comment:
There are several instances of "shortcut". I don't think the semantics of shortcut are exactly the same. To me shortcut means "take a shortcut, skip some things", which involves _not doing something_.